tests/php/Model/SiteTreeTest.php

```diff
@@ -1634,36 +1672,13 @@ public function testLinkExtension()
     }
 
     /**
-     * Test that the controller name for a SiteTree instance can be gathered by appending "Controller" to the SiteTree
+     * Test that the controller name for a Page instance can be gathered by appending "Controller" to the Page
      * class name in a PSR-2 compliant manner.
      */
     public function testGetControllerName()
     {
-        $class = new Page;
```

Comment:
Why is all of this being removed?

Reply:
Looks like a mistake when I was resolving conflicts. Will add back in.

Reply:
Done